release.sh

```diff
@@ -28,9 +34,17 @@ if [[ ! $GIT_BRANCH =~ main ]]; then
     exit 1
 fi
 
-echo Releasing version "$VERSION"
+RELEASE_BRANCH="temp-release-branch-${VERSION_PREFIXED}"
+git branch "$RELEASE_BRANCH"
+git checkout "$RELEASE_BRANCH"
```

Comment:
Are we sure at this point that we are branching from an up to date main?

Reply:
maybe we can just add a 'git pull' - if that fails, because you have uncommitted files lying around, that's totally what we want, isn't it?

Reply:
good point - we can add this